src/test/java/com/simplymadeapps/simple_logger_android/TestLogs.java

```diff
@@ -377,33 +381,30 @@ public void onFailure(Exception exception, int successful_uploads, int unsuccess
             }
         };
 
-        List<RecordedLog> list_of_logs1 = new ArrayList<>();
-        RecordedLog log1 = new RecordedLog();
-        log1.setDate(System.currentTimeMillis());
-        list_of_logs1.add(log1);
+        String today_title = SimpleAmazonLogs.date_format_title.format(new Date(System.currentTimeMillis()));
+        File file1 = mock(File.class);
+        doReturn(today_title).when(file1).getName();
 
-        List<RecordedLog> list_of_logs2 = new ArrayList<>();
-        RecordedLog log2 = new RecordedLog();
-        log2.setDate(System.currentTimeMillis()-SimpleAmazonLogs.HRS_24_IN_MS);
-        list_of_logs2.add(log2);
+        File file2 = mock(File.class);
+        doReturn("").when(file2).getName();
 
         SimpleAmazonLogs.unsuccessful_calls = 0;
         SimpleAmazonLogs.successful_calls = 0;
 
-        SimpleAmazonLogs.getTransferListener(2, list_of_logs1, file, callback).onStateChanged(0, ts);
+        SimpleAmazonLogs.getTransferListener(2, file1, callback).onStateChanged(0, ts);
         waitForThread(sleep_time_short);
 
         SimpleAmazonLogs.unsuccessful_calls = 0;
         SimpleAmazonLogs.successful_calls = 1;
 
-        SimpleAmazonLogs.getTransferListener(2, list_of_logs2, file, callback).onStateChanged(0, ts);
+        SimpleAmazonLogs.getTransferListener(2, file2, callback).onStateChanged(0, ts);
         waitForThread(sleep_time_short);
 
         SimpleAmazonLogs.unsuccessful_calls = 0;
         SimpleAmazonLogs.successful_calls = 2;
 
-        verify(file, times(2)).delete();
-        verify(dao, times(1)).delete(any(List.class));
+        verify(file1, times(0)).delete();
+        verify(file2, times(1)).delete();
 
         SystemClock.sleep(1000);
```

Comment:
Can we sleep a lot less for these tests? 1 second seems excessive. Should reduce this as much as we can across the board.

Reply:
I have set them all to 100ms instead of 1000ms.  The only reason it is so high is because the tests would randomly fail with lower values.  This might not be the case now with Jenkins and text files instead of the database.  I will leave it at 100ms and if my tests start failing I will reassess.